### PR TITLE
TooMuchStress

### PR DESCRIPTION
**Pull request by BB9653**
TMS CANOpen Object Dictionary upgrade.

### CODE REVIEW

File: include/TMS/TMS.hpp [hunk 1]
```diff
@@ -91,214 +93,41 @@ class TMS {
...
-        },
+        MANDATORY_IDENTIFICATION_ENTRIES_1000_1014,
+        HEARTBEAT_PRODUCER_1017(2000),
+        IDENTITY_OBJECT_1018,
+        SDO_CONFIGURATION_1200,// Mandatory Keys
+
+        TRANSMIT_PDO_SETTINGS_OBJECT_18XX(0, 0, NODE_ID, TRANSMIT_PDO_TRIGGER_TIMER),
+        TRANSMIT_PDO_SETTINGS_OBJECT_18XX(1, 1, NODE_ID, TRANSMIT_PDO_TRIGGER_TIMER),
+
+        TRANSMIT_PDO_MAPPING_START_KEY_1AXX(0, 4),
```
Comment (mjmagee991) [inline include/TMS/TMS.hpp:99]: Add comments to the dictionary like the examples to make it a bit more clear

File: targets/TMS/main.cpp [hunk 2]
```diff
@@ -111,7 +75,7 @@ int main() {
 
     // Initialize CAN, add an IRQ that will populate the above queue
     IO::CAN& can = IO::getCAN<IO::Pin::PA_12, IO::Pin::PA_11>();
-    can.addIRQHandler(canInterruptHandler, reinterpret_cast<void*>(&canOpenQueue));
+    can.addIRQHandler(canInterrupt, reinterpret_cast<void*>(&canOpenQueue));
 
     // Initialize the timer
     DEV::Timer& timer = DEV::getTimer<DEV::MCUTimer::Timer16>(100);
```
Comment (mjmagee991) [inline targets/TMS/main.cpp:80]: There is some other stuff in this function that should be updated to the new standard. Don't worry about the stuff in the while loop just yet

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Two small details

**Review by mjmagee991 — APPROVED** [review]
Nice work!

**Review by ActuallyTaylor — APPROVED** [review]
Solid CAN

**Pull request merged**